src/components/comment.jsx

```diff
@@ -0,0 +1,144 @@
+import React, { useState } from 'react';
+import Edit from './edit';
+import Delete from './delete'
+
+const Comment = (props) => {
+  const [replyClicked, setReplyClicked] = useState(false);
+  const [reply, setReply] = useState('');
+  const [comments, setComments] = useState(props.comments);
+
+  const handleReplySubmit = (e) => {
+    e.preventDefault();
+    if (reply === '') {
+      return;
+    }
+
+    const replyData = {
+      id: 0,
+      content: reply,
+      createdAt: 'now',
+      score: 0,
+      replyingTo: '',
```

Comment:
Can we work out how to add this data and the below for the user?

Reply:
this has been updated now